code/lib/builder-vite/src/codegen-iframe-script.ts

```diff
@@ -27,9 +29,7 @@ export async function generateIframeScriptCode(options: ExtendedOptions) {
     // Ensure that the client API is initialized by the framework before any other iframe code
     // is loaded. That way our client-apis can assume the existence of the API+store
     import { configure } from '${rendererName}';
-
-    import * as clientApi from "@storybook/client-api";
-    import { logger } from '@storybook/client-logger';
+    import { clientApi } from '${frameworkName}';
```

Comment:
SSv6 with the vite-builder and pnpm has been broken for a while.  I figured I might as well fix it in this PR, but I can move it to a different one if preferred.  I'd like to make a separate PR soon to create a `vite-core` that each of the frameworks re-export from, to avoid all the duplication in the frameworks above.  I'll do that soon.